pkgs/applications/networking/mailreaders/mmh/default.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv, fetchurl, ncurses, autoconf, automake, flex }:
+stdenv.mkDerivation rec {
+  pname = "mmh-unstable";
+  version = "2019-09-08";
+  buildInputs = [ ncurses autoconf automake flex ];
```

Comment:
`autoconf`, `automake` and `flex` usually go to `nativeBuildInputs`. You could use `autoreconfHook` instead (it also goes to `nativeBuildInputs`).